sdk/servicebus/service-bus/src/index.ts

```diff
@@ -17,13 +17,15 @@ export {
 export { QueueClient } from "./queueClient";
 export { TopicClient } from "./topicClient";
 export { SubscriptionClient } from "./subscriptionClient";
+export { Client } from "./client";
 
 export { Sender } from "./sender";
 export { Receiver, SessionReceiver } from "./receiver";
 
 export { MessageHandlerOptions } from "./core/streamingReceiver";
 export { OnError, OnMessage } from "./core/messageReceiver";
 export { SessionReceiverOptions, SessionMessageHandlerOptions } from "./session/messageSession";
+export { ReceivedMessage } from "./serviceBusMessage";
```

Comment:
Doesn't this force us to export `ReceivedMessageInfo` as well?

Reply:
Probably. I'll go ahead and remove this fix and the one you mentioned above so it's only moving us to api-extractor 7 but has no code fixes as part of it.